apps/passport-server/src/apis/pretix/genericPretixAPI.ts

```diff
@@ -422,10 +438,10 @@ export class GenericPretixAPI implements IGenericPretixAPI {
     });
   }
 
-  // Fetch all check-in lists for a given event.
+  // TODO: @rob - what's your best description of what's going on here?
```

Comment:
When checking in, it is necessary to provide one or more checkin list IDs: https://docs.pretix.eu/en/latest/api/resources/checkin.html

Events can have multiple checkin lists, though we tell users to set up only one, and we throw an error if anything other than one checkin list is found (in `validateEventData` in the Pretix pipeline).

When checking in, we use the checkin list ID of the sole checkin list.